pkg/sqlite/load.go

```diff
@@ -454,74 +449,59 @@ func (s *SQLLoader) getCSV(tx *sql.Tx, csvName string) (*registry.ClusterService
 	return csv, nil
 }
 
-func (s *SQLLoader) addAPIs(tx *sql.Tx, csv *registry.ClusterServiceVersion, channelEntryId int64) error {
+func (s *SQLLoader) addAPIs(tx *sql.Tx, bundle *registry.Bundle) error {
+	if bundle.Name == "" {
+		return fmt.Errorf("cannot add apis for bundle with no name: %#v", bundle)
+	}
 	addAPI, err := tx.Prepare("insert or replace into api(group_name, version, kind, plural) values(?, ?, ?, ?)")
 	if err != nil {
 		return err
 	}
 	defer addAPI.Close()
 
-	addAPIProvider, err := tx.Prepare("insert into api_provider(group_name, version, kind, channel_entry_id) values(?, ?, ?, ?)")
+	addApiProvider, err := tx.Prepare("insert into api_provider(group_name, version, kind, operatorbundle_name, operatorbundle_version, operatorbundle_path) values(?, ?, ?, ?, ?, ?)")
```

Comment:
Good question - the graph loader assumes the combination of these three fields is unique, so I thought it would be good to match those assumptions here. I think it will come down to what "operatorbundle_name" means when we don't have CSVs in the bundle.